settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't mind adding this file. Is that a Visual Studio Code settings? Is it safe? (ie. I don't want to override user-specific settings or force a specific settings)

Reply:
The settings are defaults, if you use the javascript standard lint rules. However, I would not merge this, since other users may have different settings

Reply:
Ok, in this case can you please remove it?